src/api/views.py

```diff
@@ -18,8 +21,11 @@
     " For example, if you want www.city.gov, you would enter “city”"
     " (without the quotes).",
     "extra_dots": "Enter the .gov domain you want without any periods.",
-    "unavailable": "That domain isn’t available. Try entering another one."
-    " Contact us if you need help coming up with a domain.",
+    "unavailable": mark_safe(  # nosec
```

Comment:
**suggestion (non-blocking)**: I was eventually able to convince myself that no user-input could get into this `mark_safe` through the `public_site_url()` function, but it wasn't totally obvious. How about a longer comment here above the `# nosec` line explaining why we think this is safe.